src/components/Calendar/Calendar.js

```diff
@@ -27,10 +40,8 @@ const localizer = dateFnsLocalizer({
   locales,
 });
 
-//drag and drop
```

Comment:
I wonder if we should leave this comment in and maybe even expand on it. Something along the lines of, "drag and drop not currently implemented due to bug"

Reply:
I agree with Pete, I think leaving a comment regarding the drag and drop here would be helpful for future developers.